htdocs/product/card.php

```diff
@@ -1793,6 +1801,12 @@
 				print '</tr>';
 			}
 
+			// Traceability
+			if ( empty($conf->global->PRODUCT_DISABLE_TRACEABILITY)) {
```

Comment:
I would invert the test.
Because all the features to male "control quality" can be a complete set of features with major change and a lot of things.
So i suggest to add an option PRODUCT_LOT_ENABLE_QUALITY_CONTROL into setup of module Product "Enable control quality features". This option can be set into the setup of module "Lot / serial" because i don't see how you can do this without managing lot/serial. Can you add this option and use it here in place of PRODUCT_DISABLE_TRACEABILITY ?

Reply:
I certainly can (and I will). Although, as it is not strictly "quality", may I still name it TRACEABILITY ?